build.sbt

```diff
@@ -21,8 +21,11 @@ Global / scalaVersion      := Dependencies.ScalaVersion
 Global / semanticdbEnabled := true
 Global / semanticdbVersion := scalafixSemanticdb.revision
 
-val gitCommit  = ("git rev-parse HEAD" !!).trim
-val gitBranch  = Option("git rev-parse --abbrev-ref HEAD" !!).map(_.trim).filter(_ != "main").map(_.replace('/', '-'))
+val gitCommit = ("git rev-parse HEAD" !!).trim
+val gitBranch = Option("git rev-parse --abbrev-ref HEAD" !!)
+  .map(_.trim)
+  .filter(b => b != "main" || b != "HEAD")
+  .map(_.replace('/', '-'))
```

Comment:
Just to make sure, the hyphen at the end will be also not presented? So the final release tag should look like "v30.0.0`

Reply:
correct: https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#replace(char,%20char)